plugins/outputs/http/README.md

```diff
@@ -0,0 +1,34 @@
+# HTTP Output Plugin
+
+This plugin writes to a HTTP Server using the `POST Method`.
+
+Data collected from telegraf is sent in the Request Body.
+
+### Configuration:
+
+```toml
+# Send telegraf metrics to HTTP Server(s)
+[[outputs.http]]
+  ## It requires a url name.
+  ## Will be transmitted telegraf metrics to the HTTP Server using the below URL.
+  ## Note that not support the HTTPS.
+  url = "http://127.0.0.1:8080/metric"
+  ## http_headers option can add a custom header to the request.
+  ## The value is written as a delimiter(:).
+  ## Content-Type is required http header in http plugin.
+  ## so content-type of HTTP specification (plain/text, application/json, etc...) must be filled out.
+  http_headers = [ "Content-Type:application/json" ]
+  ## With this HTTP status code, the http plugin checks that the HTTP request is completed normally.
+  ## As a result, any status code that is not a specified status code is considered to be an error condition and processed.
+  expected_status_codes = [ 200, 204 ]
```

Comment:
Let's call this `success_status_codes` and include 201 in the default list.

Reply:
I got it. I have to add the 201 status code and rename expected_status_codes => success_status_codes.

Reply:
I have changed the code like below.
`if resp.StatusCode < 200 || resp.StatusCode > 209 {
    return fmt.Errorf("received bad status code, %d\n", resp.StatusCode)
}`